authorization/api-secret-key/README.md

```diff
@@ -0,0 +1,62 @@
+# Tapsi Pack API: API Secret Key Guide
```

Comment:
I guess it would be useful if we could have a diagram flow in this doc for better understanding like what we have in the OAuth doc.

Reply:
I think it's best to first draw this diagram together on paper or whiteboard before adding a diagram to the documentation